cmd/createorupdate/update.go

```diff
@@ -23,6 +23,15 @@ func update(ctx context.Context, cs *api.OpenShiftManagedCluster, p api.Plugin)
 	vmc := compute.NewVirtualMachineScaleSetVMsClient(cs.Properties.AzProfile.SubscriptionID)
 	vmc.Authorizer = authorizer
 
+	bm := needUpdate(cs, oldCs)
+	if len(bm) == 1 && bm["scale"] {
```

Comment:
I don't think this is right yet because needUpdate is only partially implemented.  It should fire only if nothing but counts (or a single count?) has changed.  Currently it would fire if a count + 0 or more additional things had changed.

Reply:
Right, this needs more work